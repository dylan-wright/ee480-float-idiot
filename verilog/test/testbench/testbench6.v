
    reg `WORD x,y;
    reg `OP op;
    wire `WORD z;
    alu uut(z,op,x,y);

    initial begin
op = `OPinvf;
x = 16'h3f80;
#10;
$display("%h", z);
x = 16'h4000;
#10;
$display("%h", z);
x = 16'h4040;
#10;
$display("%h", z);
x = 16'hbf80;
#10;
$display("%h", z);
x = 16'hc2f0;
#10;
$display("%h", z);
