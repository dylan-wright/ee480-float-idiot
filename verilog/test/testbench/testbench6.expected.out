3f80
3f00
3eaa
bf80
bc08
